tools/cxbe/Common.cpp

```diff
@@ -1,6 +1,7 @@
-// Licensed under GPLv2 or (at your option) any later version.
-// Copyright (C) 2019 Jannik Vogel
-// Copyright (C) 2002-2003 Aaron Robinson <caustik@caustik.com>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// SPDX-FileCopyrightText: 2002-2003 Aaron Robinson <caustik@caustik.com>
+// SPDX-FileCopyrightText: 2019 Jannik Vogel
```

Comment:
Not sure how easy it is, but feel free to put my changes under MIT license (or even CC0)

Reply:
I'll keep that in mind, but since this code isn't included in xbes I don't think I'll go for relicensing cxbe.